docker/quantum/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7
+#FROM continuumio/miniconda3:4.8.2 as build
+
+# Switch back to root, set up the environment.
+#USER 0
+#ENV PATH /opt/conda/bin/:$PATH
+
+COPY requirements.txt /root/
+
+# Update apt, install blas/lapack for psi4, then Python deps, and then build.
+RUN apt-get update && \
+  apt-get install -y libblas-dev liblapack-dev && \
+  cd /root && \
+  pip install -r requirements.txt && \
+  wget https://github.com/psi4/psi4/archive/v1.3.2.tar.gz && \
+  tar zxvf v1.3.2.tar.gz && \
+  mkdir build && \
+  cd build && \
+  cmake ../psi4-1.3.2 && make -j10 install && \
+  rm -rf /usr/local/psi4/share/psi4/samples && \
```

Comment:
I tried the psi4 inside the image and it's failing with the following: 

```
  root@2fc7f25c76b0:/tmp# /usr/local/psi4/bin/psi4 input.dat
Traceback (most recent call last):
  File "/usr/local/psi4/bin/psi4", line 287, in <module>
    exec(content)
  File "<string>", line 35, in <module>
  File "/usr/local/psi4/lib/psi4/driver/driver.py", line 556, in energy
    wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
  File "/usr/local/psi4/lib/psi4/driver/procrouting/proc.py", line 2002, in run_scf
    scf_wfn = scf_helper(name, post_scf=False, **kwargs)
  File "/usr/local/psi4/lib/psi4/driver/procrouting/proc.py", line 1276, in scf_helper
    base_wfn = core.Wavefunction.build(scf_molecule, core.get_global_option('BASIS'))
  File "/usr/local/psi4/lib/psi4/driver/p4util/python_helpers.py", line 115, in _core_wavefunction_build
    basis = core.BasisSet.build(mol, "ORBITAL", basis)
  File "/usr/local/psi4/lib/psi4/driver/p4util/python_helpers.py", line 80, in _pybuild_basis
    mol.to_dict(), key, resolved_target, fitrole, other, return_dict=True, return_atomlist=return_atomlist)
  File "/usr/local/psi4/lib/psi4/driver/qcdb/molecule.py", line 1461, in to_dict
    compare_molrecs(validated_molrec, molrec, 6, 'to_dict', forgive=forgive, verbose=0)
  File "/usr/local/psi4/lib/psi4/driver/qcdb/psiutil.py", line 255, in compare_molrecs
    compare_dicts(xptd, cptd, tol, label, forgive=forgive, verbose=verbose)
  File "/usr/local/psi4/lib/psi4/driver/qcdb/psiutil.py", line 152, in compare_dicts
    raise ImportError("""Python module deepdiff not found. Solve by installing it: `conda install deepdiff -c conda-forge` or `pip install deepdiff`""")

ImportError: Python module deepdiff not found. Solve by installing it: `conda install deepdiff -c conda-forge` or `pip install deepdiff`

Printing out the relevant lines from the Psithon --> Python processed input file:
    ""","cswater")
    core.IO.set_default_namespace("cswater")
    oswater = cswater.clone()
    oswater.set_multiplicity(2)
    oswater.set_molecular_charge(1)
--> e = energy('b3lyp', molecule=cswater)
    core.set_global_option("REFERENCE", "uhf")
    e = energy('b3lyp', molecule=oswater)
    core.set_global_option("REFERENCE", "rks")
    e = energy('b3lyp5', molecule=cswater)
    core.set_global_option("REFERENCE", "uks")
``` 

The example I was trying to run is: https://github.com/psi4/psi4/blob/master/samples/dft-b3lyp/input.dat